sim/mcuboot-sys/src/area.rs

```diff
@@ -176,11 +176,8 @@ pub enum FlashId {
     Image0 = 1,
     Image1 = 2,
     ImageScratch = 3,
-    Nffs = 4,
-    Core = 5,
-    RebootLog = 6,
-    Image2 = 7,
-    Image3 = 8,
+    Image2 = 4,
+    Image3 = 5,
```

Comment:
If you had to do this, I assume the prev commit (a15e8f8ca297f4aab787b7b9374fd7f87d2606dc) so why not just squash there? Also would not it be better to make it:
```
    Image0 = 1,
    Image1 = 2,
    ImageScratch = 3,
    Image2 = 4,
    Image3 = 5,
    Nffs = 6,
    Core = 7,
    RebootLog = 8,
```

Reply:
Is there any reason to have the extra areas that we don't use?

Reply:
Whatever about the extra areas, does it work like this?
```
    Image0 = 1,
    Image1 = 2,
    ImageScratch = 3,
    Nffs = 4,
    Core = 5,
    RebootLog = 6,
    Image2 = 7,
    Image3 = 8,
```
If it does not, you are fixing a broken previous commit, so squash...

Reply:
Ok, yeah, I'll squash it.